src/api/app/models/bs_request.rb

```diff
@@ -1260,7 +1260,6 @@ def self.extend_query_for_project(requests, roles, states, review_states, packag
 
   def self.extend_relation(source_or_target, requests, roles, package, subprojects, project, inner_or)
     if roles.empty? || roles.include?(source_or_target)
-      requests = requests.references(:bs_request_actions)
```

Comment:
Not sure if you can remove it here as requests gets called in BsRequest.collection (1143) with requests = with_actions, so actually it does join with bs_request_actions